Modules/GLO/include/GLO/Helpers.h

```diff
@@ -100,8 +100,7 @@ struct K0sFitter {
     mSignalAndBackground->SetParameter(Parameters::Mass, mMassK0s);
     mSignalAndBackground->SetParameter(Parameters::Sigma, 0.005);
     mSignalAndBackground->SetParLimits(Parameters::Sigma, 1e-6, 1);
-    h->Fit(mSignalAndBackground.get(), (add) ? "RMQ" : "RMQ0");
-    return true;
+    return h->Fit(mSignalAndBackground.get(), (add) ? "RMQ" : "RMQ0") == 0;
```

Comment:
I don't think that TH1::Fit returns a boolean. What do you think ?

Reply:
You're right.